### PR TITLE
Fix sending buffer in MySigning.cpp

### PR DESCRIPTION
**Pull request by tekka007**

### CODE REVIEW

**Comment by tekka007** [comment]
Jenkins, build examples please

File: libraries/MySensors/core/MySigning.cpp [hunk 1]
```diff
@@ -106,6 +106,7 @@ static void prepareSigningPresentation(MyMessage &msg, uint8_t destination) {
 	// Only supports version 1 for now
 	build(msg, _nc.nodeId, destination, NODE_SENSOR_ID, C_INTERNAL, I_SIGNING_PRESENTATION, false).set("");
 	mSetLength(msg, 2);
+	mSetPayloadType(msg, P_CUSTOM);		// displayed as hex
```
Comment (fallberg) [inline libraries/MySensors/core/MySigning.cpp]: Perhaps this can be a argument to build() if it is not already available?

Comment (tekka007) [inline libraries/MySensors/core/MySigning.cpp]: No it's not, and I don't see many use cases for that.

**Comment by tekka007** [comment]
Jenkins, build examples please

**Pull request merged**